File: DotNet_Fundamentals/CLS_ConsumeNonCompliantCSharpCode/Class1.vb
Imports CLS_NonCompliantCSharpCode
Public Class MyVbClass
    Public Sub Check()
        Dim obj As New Class1()
        'Assembly violates CLS rule of case sensitive
        'It's not possible to access variable with same name but only change in cases
        'obj.inta;'possible
        'obj.intA;'not possible
        'obj.intx; 'possible

    End Sub
End Class

File: DotNet_Fundamentals/CLS_NonCompliantCSharpCode/Class1.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace CLS_NonCompliantCSharpCode
{
    public class Class1
    {
        //C# is case sensitive, It's possible to access variable with same name but only change in cases
        //To check code is CLS compliant or not use, [assembly: CLSCompliant(true)] in AssemblyInfo file 
        public int intA;
        public int inta;
        public int intx;
    }
}
